domain/user.js

```diff
@@ -0,0 +1,162 @@
+class User {
+  constructor(userId, name) {
+    this.id = userId;
+    this.name = name;
+  }
+
+  viewProfile() {
+    throw new Error(`${this.viewProfile.name} is not implemented.`);
+  }
+
+  updateProfile() {
+    throw new Error(`${this.updateProfile.name} is not implemented.`);
+  }
+
+}
+
+class UserId {
```

Comment:
Isn't this UserId class a bit overkill since the uid will be a simple string from firebase?

Reply:
Maybe, when I first created it I did not know from where we would get this.

As a side note for the future, if we do authorization later, that could be a place where we put the authorization code.

Reply:
ok, I see